packages/shopping/src/__tests__/e2e/integration/cart.spec.js

```diff
@@ -0,0 +1,64 @@
+context('Cart', () => {
+  before(() => cy.visit('http://localhost:3000/shoppy.html'));
+
+  it('should checkout all items from Cart', () => {
+    cy.get('#logIn')
+      .click()
+      .then(() => cy.get('#logInButton').click());
+
+    cy.get('#shoppingCartLink')
```

Comment:
When running this locally, it failed because it couldn't click on this:
<img width="1194" alt="Screen Shot 2020-10-21 at 9 20 53 AM" src="https://user-images.githubusercontent.com/42985749/96725270-b9444c00-137e-11eb-974e-56693eafc86e.png">

Can you verify the login modal isn't visible before this line?

Reply:
Will check to ensure its visible before executing the click